cli/src/prelude.rs

```diff
@@ -1,7 +1,8 @@
 pub(crate) use {
     crate::error::NexusCliError,
     anyhow::{anyhow, bail, Error as AnyError, Result as AnyResult},
-    clap::{builder::ValueParser, Args, Parser, Subcommand, ValueEnum},
+    clap::{builder::ValueParser, Args, CommandFactory, Parser, Subcommand, ValueEnum},
+    clap_complete,
```

Comment:
Why did you have to add `clap_complete` here? The linter says this import is redundant.

Reply:
Well, it was not redundant when I did not have it and it did not compile ... 🤷

Reply:
It seems I cannot locally reproduce what was happening that led me to add it here. It now
compiles fine without it, though my IDE complains. Removing ...

Reply:
https://github.com/Talus-Network/nexus-sdk/pull/158/commits/f8aa18b2aef05c65d283d92cdd925c11b78395a1